src/documentation/3.4.0/setup/native/ubuntu/README.md

```
@@ -171,6 +177,20 @@ sudo systemctl status mongod
 
 ```
 
+## Setting up Kafka
+
+**Start Kafka Service**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)